### DISCUSSION TITLE
use_rtti=true

### DISCUSSION THREAD

**Discussion by csagan5** [ungoogled-software/ungoogled-chromium-android#76] [Q&A]
Hi there @wchen342

I noticed that you use `use_rtti=true`, but does it compile for you on x86? It does not for me, it gives an error about the vtable

**Comment by wchen342** [comment]
What version is it? It compiles on `.182` but I haven't tried anything newer than that.

**Reply by csagan5** [reply]
It was happening with all recent versions.

**Comment by wchen342** [comment]
I ran into this today when building monochrome. It is the only time I got the error, it didn't happen with ChromeModern or TriChrome. I guess it has something to do with crazy linker.

EDIT: actually not crazy linker. Maybe `lld` problem?

**Reply by csagan5** [reply]
No idea; it happens also for ChromeModern for Bromite. Perhaps it only happens with the combination of GN flags that Bromite uses?

If you use ccache then most probably the problem is concealed by that.
